.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  test:
```

Comment:
OPT: Maybe call this "build" as it does no tests ;)

Reply:
Make sense! Edited the job name.